django/db/models/enums.py

```diff
@@ -60,7 +60,8 @@ def values(cls):
 
 class Choices(enum.Enum, metaclass=ChoicesMeta):
     """Class for creating enumerated choices."""
-    pass
+    def __str__(self):
+        return "%s" % self.value
```

Comment:
No need for formatting here.

```suggestion
        return str(self.value)
```

Reply:
Yep. That's much nicer. Thanks. 🤯